### ISSUE TITLE
Error when Bot lacks Embed Links Permission

### ISSUE THREAD

**Issue by TheBrianiac** [Frederikam/FredBoat#167]
**To Reproduce:**
- Start FredBoat playing a song.
- Remove FredBoat's embed link permissions in a channel.
- Run ;;nowplaying in that channel.

**Error:**
As it was pasted in chat:
```net.dv8tion.jda.core.exceptions.PermissionException: Cannot perform action due to a lack of Permission. Missing permission: MESSAGE_EMBED_LINKS
    net.dv8tion.jda.core.entities.impl.TextChannelImpl.checkPermission(TextChannelImpl.java:606)
    net.dv8tion.jda.core.entities.impl.TextChannelImpl.checkPermission(TextChannelImpl.java:598)
    net.dv8tion.jda.core.entities.impl.TextChannelImpl.sendMessage(TextChannelImpl.java:274)
    net.dv8tion.jda.core.entities.MessageChannel.sendMessage(MessageChannel.java:273)
    
fredboat.command.music.info.NowplayingCommand.sendYoutubeEmbed(NowplayingCommand.java:118)
    fredboat.command.music.info.NowplayingCommand.onInvoke(NowplayingCommand.java:67)
    fredboat.commandmeta.CommandManager.prefixCalled(CommandManager.java:118)
```

For questions, I'm on Discord, Otis#9699

**Comment by napstr** [comment]
Solution: Do not revoke FredBoat's embed permissions.
???
Profit

**Comment by shikhir-arora** [comment]
This makes sense, though. YouTube `;;nowplaying` is outputted with an embed, so it needs that permission. The error is from JDA telling you just that, to check the permissions 👍

**Comment by TheBrianiac** [comment]
It should still be handled as an expected error and get an English message ("Hey, I need permissions" for somebody who doesn't speak Java stack trace), or just post without the embed link permission (information besides the link can still be relevant).

**Comment by Frederikam** [comment]
Agreed, this should send a localised error message instead of a stack trace.

**Comment by knight-ryu12** [comment]
Just change few lines
and you are good to go

**Comment by Frederikam** [comment]
Hence the beginner label 😛

**Comment by napstr** [comment]
Fixed here: 8d7427d58034f58f20ea371d914b15acd6ec1ea3

Long term fix a whole class of these problems:

![image](https://cloud.githubusercontent.com/assets/6048348/25587486/942633d8-2ea4-11e7-95e8-195af95d99b3.png)

@Frederikam please set those bot invite links up with what you deem acceptable.
Here's the site: https://discordapi.com/permissions.html
